### PR TITLE
fixed warnings and simplified Github Actions

### PR DESCRIPTION
**Pull request by s-martin**
* fixed flake8 formatting warnings
* Github Actions only for Python 3.7

### CODE REVIEW

**Comment by MiczFlor** [comment]
Hi @s-martin 
you changed the tabs in the python code. Does this run on your Pi?

**Comment by ZyanKLee** [comment]
> Hi @s-martin
> you changed the tabs in the python code. Does this run on your Pi?

hello! to avoid problems with indentation differences in the future I just opened pr #742 introducing a suggestion for an editorconfig file

File: scripts/python-phoniebox/PhonieboxConfigChanger.py [hunk 1]
```diff
@@ -34,47 +37,47 @@ def str2num(s):
...
         try:
-            parser.set(section,key,value)
-            self.debug("Set {} = {} in section {}".format(key,value,section))
+            parser.set(section, key, value)
+            self.debug("Set {} = {} in section {}".format(key, value, section))
         except configparser.NoSectionError as e:
             raise(configparser.NoSectionError, e)
         # write to file
 #        with open(config_file, 'w') as f:
 #            parser.write(f)
```
Comment (ZyanKLee) [inline scripts/python-phoniebox/PhonieboxConfigChanger.py]:
```suggestion:-1+0
        # with open(config_file, 'w') as f:
        #     parser.write(f)
```

**Comment by ZyanKLee** [comment]
aside from that tiny nitpick, the code is now cleanly aligned - means it should run and it is a miracle it did before :laughing:

**Comment by s-martin** [comment]
> 
> 
> Hi @s-martin
> you changed the tabs in the python code. Does this run on your Pi?

I did not have a chance to test it on my Pi yet. It worked in my Ubuntu VM at least for the non-Pi specific parts.

File: scripts/python-phoniebox/PhonieboxConfigChanger.py (hunk 1 above)
Comment (s-martin) [reply scripts/python-phoniebox/PhonieboxConfigChanger.py]: Fixed now

**Pull request merged**